helpers/sql-obfuscation/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: opentelemetry-helpers-sql-obfuscation

```

Comment:
For initial Gem releases; only this line should be included. Below lines should be removed; since it will be added by the release tooling

Reply:
Good catch! Fixed in: bb37c867f04e2e011939eea9ba8f92c1e1258db6